pkg/v1/tkg/test/config/aws.yaml

```diff
@@ -19,3 +19,5 @@ tkg_config_variables:
   AWS_PRIVATE_SUBNET_ID_2: subnet-016eb6a45203c010e
 management_cluster_options:
   plan: dev
+workload_cluster_options:
+  CLUSTER_CLASS_CB_FILE_PATH: ../../../test/data/testdata/customClusterBootstrapAntrea-aws.yaml
```

Comment:
Shouldn't this be ```../data/testdata/customClusterBootstrapAntrea-aws.yaml```, as these two files are both under ```test``` already and its only their subdirectories ```config``` vs ```data``` is different

Reply:
updated this file path